_posts/2021-02-12-generalist-specialization-and-the-impostor-syndrome.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Generalist Specialization and The Impostor Syndrome
+author: arto
+excerpt: >
+  Impostor Syndrome is a common visitor in our profession, as we are often in charge of shaping the world of the future, while boldly going to new, unknown areas. In this article I'll touch on what it is, how to recognize it, what to do about it, and how does it relate to being a generalist.
+tags:
+ - Software Development
+ - Impostor syndrome
+---
+
+![Feeling of self-doubt](/img/impostor-syndrome/doubt.jpg)
+
+_I don't know how to do this. This is all too overwhelming. Why did I want to do this? Everybody else knows what they are doing. They will soon discover how out of my league I am here._
+
+Sounds familiar? Maybe not. Not everyone is suffering from Impostor Syndrome. But it is quite common in our field of work. I've had those thoughts myself, and many times have also seen others go through moments like this. It's not typically a permanent state in life, but join me for a bit and let's talk about it.
+
+## Why do we feel like impostors?
+
+Creating quality software in 2021 is hard. There are so many facets to consider in addition to just writing the code. Things like the choice of programming languages, frameworks, and databases are just the start. On-prem or cloud-native or hybrid? What and how to automate? What tools to pick for IaC?  What's the frontend framework of the month? Have you taken proper care of security? How about privacy? Did you think about accessibility? And finally, how is the project going? Communication working? Doing the right things in the right order?
+
+Neglecting any of these, in a professionally done project, can doom it to failure, or at least make it very difficult. So to be able to do your work properly, there's an endless stream of things you need to be aware of and make the right choices. The more you know the more you know what you don't know.
+
+Furthermore, things move rapidly. While you can use your knowledge from decades back, in our line of work we typically need to be constantly learning more, keeping skills up to date, replacing old tools with new, better ones. While at some point in time you could say you know everything there is about programming, now things have exploded exponentially. If anyone tells you they know all there is to know about software delivery, they are lying.
+
+So, that's the scene. That's the setup. Things might be a bit calmer if you're working on the same product/service for years, but I can tell you that especially in consulting business, you have to push yourself to new areas constantly, and quickly master them. Many people actually enjoy that, on a good day it's a rush to be able to grow and learn new things.
```

Comment:
"if you work on"

"in *the* consulting business"

"… enjoy that. On a good…"